prow/github/client.go

```diff
@@ -530,6 +530,17 @@ func (c *Client) CloseIssue(org, repo string, number int) error {
 	return err
 }
 
+func (c *Client) OpenIssue(org, repo string, number int) error {
```

Comment:
total nit, but avoids confusion.  Since this isn't opening an issue (which hits a different [endpoint](https://developer.github.com/v3/issues/#create-an-issue)), can we call the func ReopenIssue or something that makes it clear that this only works for existing, closed issues.  Also, please add a comment